template/azurePipelines/kustomize/.pipelines/azure-kubernetes-service-kustomize.yaml

```diff
@@ -0,0 +1,68 @@
+# Azure Kubernetes Service (AKS) pipeline with Kustomize
+# Build and push image to Azure Container Registry; Deploy to Azure Kubernetes Service cluster
+
+variables:
+  armServiceConnection: {{ARM_SERVICE_CONNECTION}}

```

Comment:
this seems like a weird name for what's the azure subscription

Reply:
It's the name of the Azure Resource Manager service connection, not the subscription

Reply:
The docs and how it's used here seem to imply it's the subscription. What is the ARM Service Connection?

Reply:
https://learn.microsoft.com/en-us/azure/devops/pipelines/library/connect-to-azure?view=azure-devops#create-service-connection-with-workload-identity-federation-manual

This is what we'll be using.
I know it says Azure subscription in the [KubernetesManifest@1](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/kubernetes-manifest-v1?view=azure-pipelines) doc, but it does not take the subscription ID or name and expects the service connection name.

Reply:
Makes sense to me!